<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton_closeWin">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>810</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭窗口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imgOri">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>471</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string>原始图像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imgPro">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>471</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string>处理图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_nextImg">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>780</y>
     <width>145</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>播放/暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_camAdr">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>视频地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bro">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_playSpeed">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>680</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>播放速度</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_playSpeed">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>680</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_closeWin</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
